aasemble/django/apps/buildsvc/tests_functional.py

```diff
@@ -47,3 +49,109 @@ def test_secured_pages_open_after_login(self):
         page_header = self.selenium.find_element_by_class_name('page-header')
         text_found = re.search(r'Sources', page_header.text)
         self.assertNotEqual(text_found, None)
+
+    def test_source_package(self):
+        '''This test performs a basic package addition and deletion.
+           This test consists of following steps:
+           1. Create a session cookie for given user. This step includes a
+              user creation.
+           2. Create a group. This will serve as 'Extra admin' for test repo.
+           3. Create a test repo under user that we have created in step 1
+           4. Create a series with the repo of step 3.
+           5. Try to create a package.
+           6. Verify if the package has been created.
+           7. Try to delete the package
+           8. Verify if the package has been deleted
+           9. Peform cleanup like delete user. repo etc that we have created.'''
+        session_cookie = create_session_cookie(username='myuser', password='123456')
```

Comment:
Now that the fixtures contain a number of users, let's not add more tests that create their own users.

In other words: Please use one of the existing users defined in aasemble/django/apps/api/fixtures/complete.json. See  docs/Testing.md for a description of each user.